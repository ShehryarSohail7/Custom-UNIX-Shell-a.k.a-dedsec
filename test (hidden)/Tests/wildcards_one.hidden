find . -name *.hidden
ls *.txt
ls *f*
ls ????.*
ls ????.log
ls *f*.* | grep .log