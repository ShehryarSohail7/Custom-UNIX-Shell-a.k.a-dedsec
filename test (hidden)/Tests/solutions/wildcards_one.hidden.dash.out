./Tests/builtins_one.hidden
./Tests/builtins_two.hidden
./Tests/chaining.hidden
./Tests/exec_only.hidden
./Tests/ioredir_one.hidden
./Tests/ioredir_two.hidden
./Tests/pipeline_ioredir.hidden
./Tests/pipeline_one.hidden
./Tests/pipeline_two.hidden
./Tests/wildcards_one.hidden
./Tests/wild_chaining.hidden
names.txt
config.json
find.log
find.log
test.py
find.log
find.log
